packages/react/src/global-alert/GlobalAlert.tsx

```diff
@@ -1,63 +1,119 @@
-import { MouseEventHandler, PropsWithChildren } from 'react';
+import { MouseEventHandler, PropsWithChildren, ReactNode } from 'react';
 import { Box } from '../box';
 import { Flex } from '../flex';
 import { Stack } from '../stack';
-import { Button } from '../button';
-import { boxPalette } from '../core';
-import { CloseIcon, WarningFilledIcon } from '../icon';
-import { Content } from '../content';
+import { boxPalette, tokens, mapSpacing } from '../core';
+import { WarningFilledIcon, InfoFilledIcon } from '../icon';
 import { Heading } from '../heading';
+import {
+	GlobalAlertCloseButton,
+	GLOBAL_ALERT_CLOSE_BUTTON_LABEL_VISIBLE_MQ,
+} from './GlobalAlertCloseButton';
+
+type GlobalAlertTone = 'info' | 'warning';
 
 export type GlobalAlertProps = PropsWithChildren<{
 	/** Function to be called when the 'Close' button is pressed. */
 	onDismiss?: MouseEventHandler<HTMLButtonElement>;
 	/** The title of the alert. */
 	title?: string;
+	/** The tone of the alert. */
+	tone?: GlobalAlertTone;
 }>;
 
-export function GlobalAlert({ children, onDismiss, title }: GlobalAlertProps) {
+const toneMap = {
+	info: {
+		bg: boxPalette.systemInfoMuted,
+		fg: boxPalette.systemInfo,
+		Icon: InfoFilledIcon,
+	},
+	warning: {
+		bg: boxPalette.systemWarningMuted,
+		fg: boxPalette.systemWarning,
+		Icon: WarningFilledIcon,
+	},
+};
+
+export function GlobalAlert({
+	children,
+	onDismiss,
+	title,
+	tone = 'warning',
+}: GlobalAlertProps) {
+	const { bg, Icon } = toneMap[tone];
+
 	return (
-		<section
-			aria-label={title}
-			css={{ backgroundColor: boxPalette.systemWarningMuted }}
-		>
-			<Content>
-				<Flex
-					flexDirection={['column', 'row']}
-					alignItems="flex-start"
-					justifyContent="space-between"
-					gap={[1, 1.5]}
-					paddingY={1}
-				>
-					<Flex flexDirection={['column', 'row']} gap={[0.5, 1]}>
+		<section aria-label={title} css={{ backgroundColor: bg }}>
+			<Flex
+				flexDirection="row"
+				css={{
+					position: 'relative',
+				}}
+			>
+				<GlobalAlertIconContainer tone={tone}>
+					<Icon aria-hidden="false" aria-label="Warning" color="inherit" />
+				</GlobalAlertIconContainer>
+
+				<Flex padding={1.5}>
+					<Stack gap={0.75} maxWidth={tokens.maxWidth.bodyText}>
```

Comment:
I think we should remove this `maxWidth` as it will make the heading wrap at a *really* short length. 

Can we make the consumer do this, or wrap the `children` in a `Box` with max-width.